packages/shared/src/authz/policies/project/canCreate.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { describe, expect, it } from 'vitest'
+import { canCreateProjectPolicy } from './canCreate.js'
+import { parseFeatureFlags } from '../../../environment/index.js'
+import {
+  ProjectNoAccessError,
+  ServerNoAccessError,
+  ServerNoSessionError
+} from '../../domain/authErrors.js'
+
+const buildSUT = (overrides?: Partial<Parameters<typeof canCreateProjectPolicy>[0]>) =>
```

Comment:
whats a SUT?

Reply:
System Under Test, common way to refer to the thing being tested in a unit test

Reply:
til